docs/DeveloperGuide.md

```diff
@@ -237,6 +237,27 @@ The following activity diagram summarizes what happens when a user executes a ne
   - Pros: Will use less memory (e.g. for `delete`, just save the person being deleted).
   - Cons: We must ensure that the implementation of each individual command are correct.
 
+### Add feature
+
+#### Implementation
+
+This section explains the implementation of the `add` feature. The command takes in two compulsory parameters which is the employee name and position, executing the command leads to the addition of an employee person into the records of coydir.
+
+Below is a sequence diagram and explanation of how `add` is executed.
+
+<img src="images/AddCommandUML.png" width="550" />
+
+Step 1. The user enters the command `add n/Jon j/janitor`
+
+Step 2. User input is parsed by `AddCommandParser` which creates the`AddCommand` object, then the method LogicManager#execute is called to create the AddCommand object.
```

Comment:
Perhaps can go a bit more into detail. Like AddCommandParser creates the AddCommand object while the execute function in AddCommand creates a CommandResult Object